test/tests.py

```diff
@@ -1727,10 +1727,14 @@ def test(self):
 rapidio_bank = "bank regs;"
 for dmlver in ['1.2', '1.4']:
     basedir = join(project_host_path(), "bin", "dml", dmlver, "*.dml")
-    lib_files = list(map(os.path.basename, glob.glob(basedir)))
+    lib_files = set(map(os.path.basename, glob.glob(basedir)))
+    if int(simicsutils.internal.simics_version) > 6:
+        lib_files -= {"mil-std-1553.dml", "rapidio.dml",
+                      "rapidio-device.dml"}
+
     all_tests.append(ImportTest(
         'lib-dml-%s-api-%s' % (dmlver, latest_api_version),
-        dmlver, latest_api_version, lib_files,
+        dmlver, latest_api_version, list(lib_files),
         rapidio_bank if dmlver == '1.2' else ''))
```

Comment:
the `rapidio_bank` thing is also tied to rapidio, consider adding a <=6 precondition

Reply:
Done